### ISSUE TITLE
SCC link 404

### ISSUE THREAD

**Issue by Roguito** [SUSE/suse-at-home#24]
https://github.com/SUSE/suse-at-home/blob/main/install/InstallSLESonx86.md

points to https://scc.suse.com/admin/products/1939

which is 404

**Comment by zoopster** [comment]
Good point. This would require a specific set of credentials. Would be helpful to update based on a eval or other non-subscription option.
